easybuild/easyconfigs/l/libsndfile/libsndfile-1.0.28-GCCcore-9.3.0.eb

```
@@ -0,0 +1,23 @@
+easyblock = 'ConfigureMake'
+
+name = 'libsndfile'
+version = '1.0.28'
+
+homepage = 'http://www.mega-nerd.com/libsndfile'
+description = """Libsndfile is a C library for reading and writing files containing sampled sound
+ (such as MS Windows WAV and the Apple/SGI AIFF format) through one standard library interface."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+source_urls = ['http://www.mega-nerd.com/libsndfile/files/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['1ff33929f042fa333aed1e8923aa628c3ee9e1eb85512686c55092d1e5a9dfa9']
+
+builddependencies = [('binutils', '2.34')]
+
```

Your easyconfig is missing this `configopts` from the other `libsndfile` easyconfigs in develop. Probably your branch is not up to date (this was a recent fix)
```suggestion

configopts = '--enable-octave=no'

```

---

So for some reason I'm hitting a problem:
    [centos@ip-172-31-89-45 easybuild-easyconfigs]$ git checkout bioteamMichael:20200319211001_new_pr_libsndfile1028
    error: pathspec 'bioteamMichael:20200319211001_new_pr_libsndfile1028' did not match any file(s) known to git.
I'm not quite sure what to do on this. I've even gone back and done a fresh install and checkout of everything, and still hitting this problem.

---

That branch should be in your git repo, the default `origin` in your `git remote`. You may be able to check it out with something like
```
git fetch origin
git checkout 20200319211001_new_pr_libsndfile1028
```
And to rebase it to `2020a`, in case you do not have that branch locally
```
git fetch github_easybuilders
git branch --set-upstream 2020a github_easybuilders/2020a
git checkout 20200319211001_new_pr_libsndfile1028
git rebase 2020a
```
I'm assuming that you use the _standard_ `github_easybuilders` name for the remote. Change it accordingly to your setup if it is different.

---

I'm not seeing it there -- the only thing I can think of is that I somehow didn't check in my update to my repo before submitting the PR. Can we close this one out, and then I can commit and then submit a PR the right way?